L1Trigger/L1TCalorimeter/plugins/L1TCaloRCTToUpgradeConverter.cc

```diff
@@ -100,17 +100,16 @@ L1TCaloRCTToUpgradeConverter::produce(edm::Event& iEvent, const edm::EventSetup&
     ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > p4(0,0,0,0);
 
 
-    // create new format
-    //CaloRegion region;
-    CaloRegion region(*&p4,
-			   0.,
-			   0.,
-			   (int) rgn->et(),
-			   (int) rgn->id().ieta(),
-			   (int) rgn->id().iphi(),
-			   hwQual,
-			   0,
-			   0);
+    // create new format  TODO: not filled with necessary information
```

Comment:
Nick, I think there is some misunderstanding regarding the Stage 1 definitions of Missing Et and Missing Ht. For stage 1, missing et is the vector sum of the region hwPt() values. Missing Ht is ALSO the vector sum of region hwPt() values, but only using regions above a fixed energy threshold (I think 5GeV). 

I believe also that the "phi" value associated with the mht for stage 1 is NOT the vector angle, but rather the angle between the two highest pt jets in the event. @apana should comment more on this.